sysboundary/sysboundary.cpp

```diff
@@ -164,7 +164,7 @@ void SysBoundary::initSysBoundaries(Project& project, creal& t) {
    }
 
    for (it = sysBoundaryCondList.begin(); it != sysBoundaryCondList.end(); it++) {
-      if (*it == "Outflow") {
+      if (*it == "Outflow" || *it == "outflow") {
```

Comment:
1. Make this into a `switch`
2. Just lowercase `*it`. `boost::algorithm::to_lower` or `boost::algorithm::to_lower_copy` if we want to use that or the one-liner from [cppreference](https://en.cppreference.com/w/cpp/string/byte/tolower).

Reply:
Let's not add any new boost requirements please. But we do already have a to-lowercase action in datareduction:
https://github.com/fmihpc/vlasiator/blob/f995f8eff292ee4294634e6c92545555b05bc75e/datareduction/datareducer.cpp#L48

Reply:
Yeah I figured more boost deps might be a negative. Note some sources say the implementation in datareduction is UB so see cppreference. We could just make this into a function in `common.h`.